app/src/main/java/com/group15/fitnesstracker/db/Schedule.java

```diff
@@ -1,23 +1,23 @@
 package com.group15.fitnesstracker.db;
 
 import androidx.room.Entity;
+import androidx.room.ForeignKey;
 import androidx.room.PrimaryKey;
 
-@Entity
+@Entity(foreignKeys = {@ForeignKey(entity = Workout.class,parentColumns = "workoutId",childColumns = "workoutId"),
+@ForeignKey(entity = Trainer.class,parentColumns = "trainerId",childColumns = "trainerId"),
+@ForeignKey(entity = User.class,parentColumns = "id",childColumns = "userId"),
+@ForeignKey(entity = Records.class,parentColumns = "recordId",childColumns = "recordingId")})
 public class Schedule {
     @PrimaryKey
```

Comment:
this will break, you gotta use composite primary key 

https://stackoverflow.com/a/47130788/4399065